backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -18,22 +22,28 @@ object StubRenderer {
                 }
                 +"*/"
             }
+
+            // Let's try to keep non-trivial kdoc formatting intact
+            if (shouldExportKDoc) {
+                this.descriptor?.extractKDocString()?.let { +it }
```

Comment:
Comments order is still wrong: https://github.com/JetBrains/kotlin-native/pull/4441#discussion_r543413440